### ISSUE TITLE
After screen reader shortcut is read, begin cycling audio devices

### ISSUE THREAD

**Issue by danirabbit** [elementary/installer#888]
### Problem

We can get into a bit of a chicken and egg problem where blind folks can't set the correct audio device without the screen reader, but also can't use the screen reader without having the correct audio device set.

### Proposal

After the screen reader shortcut is announced, we should begin automatically cycling available audio devices and reading the announcement again on each device

### Prior Art (Optional)

@zersiax mentioned this feature in their recent stream. I can't remember if there was an example of an installer that does this already
